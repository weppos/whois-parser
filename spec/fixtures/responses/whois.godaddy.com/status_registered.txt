Domain Name: GODADDY.COM
Registry Domain ID: 4013247_DOMAIN_COM-VRSN
Registrar WHOIS Server: whois.godaddy.com
Registrar URL: http://www.godaddy.com
Update Date: 2013-06-02T12:16:24Z
Creation Date: 1999-03-02T05:00:00Z
Registrar Registration Expiration Date: 2021-11-01T11:59:59Z
Registrar: GoDaddy.com, LLC
Registrar IANA ID: 146
Registrar Abuse Contact Email: abuse@godaddy.com
Registrar Abuse Contact Phone: +1.4806242505
Domain Status: clientTransferProhibited http://www.icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited http://www.icann.org/epp#clientUpdateProhibited
Domain Status: clientRenewProhibited http://www.icann.org/epp#clientRenewProhibited
Domain Status: clientDeleteProhibited http://www.icann.org/epp#clientDeleteProhibited
Registry Registrant ID: Not Available From Registry
Registrant Name: Domain Administrator
Registrant Organization: Go Daddy Operating Company, LLC
Registrant Street: 14455 N Hayden Rd Suite 219
Registrant City: Scottsdale
Registrant State/Province: Arizona
Registrant Postal Code: 85260
Registrant Country: US
Registrant Phone: +1.4805058800
Registrant Phone Ext: 
Registrant Fax: +1.4805058844
Registrant Fax Ext: 
Registrant Email: companynames@godaddy.com
Registry Admin ID: Not Available From Registry
Admin Name: Domain Administrator
Admin Organization: Go Daddy Operating Company, LLC
Admin Street: 14455 N Hayden Rd Suite 219
Admin City: Scottsdale
Admin State/Province: Arizona
Admin Postal Code: 85260
Admin Country: US
Admin Phone: +1.4805058800
Admin Phone Ext: 
Admin Fax: +1.4805058844
Admin Fax Ext: 
Admin Email: companynames@godaddy.com
Registry Tech ID: Not Available From Registry
Tech Name: Domain Administrator
Tech Organization: Go Daddy Operating Company, LLC
Tech Street: 14455 N Hayden Rd Suite 219
Tech City: Scottsdale
Tech State/Province: Arizona
Tech Postal Code: 85260
Tech Country: US
Tech Phone: +1.4805058800
Tech Phone Ext: 
Tech Fax: +1.4805058844
Tech Fax Ext: 
Tech Email: companynames@godaddy.com
Name Server: CNS1.GODADDY.COM
Name Server: CNS2.GODADDY.COM
Name Server: CNS3.GODADDY.COM
Name Server: A11-64.AKAM.NET
Name Server: A1-245.AKAM.NET
Name Server: A20-65.AKAM.NET
Name Server: A6-66.AKAM.NET
Name Server: A8-67.AKAM.NET
Name Server: A9-67.AKAM.NET
DNSSEC: unsigned
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net/
>>> Last update of WHOIS database: 2017-08-03T10:00:00Z <<<

For more information on Whois status codes, please visit https://www.icann.org/resources/pages/epp-status-codes-2014-06-16-en

The data contained in GoDaddy.com, LLC's WhoIs database,
while believed by the company to be reliable, is provided "as is"
with no guarantee or warranties regarding its accuracy.  This
information is provided for the sole purpose of assisting you
in obtaining information about domain name registration records.
Any use of this data for any other purpose is expressly forbidden without the prior written
permission of GoDaddy.com, LLC.  By submitting an inquiry,
you agree to these terms of usage and limitations of warranty.  In particular,
you agree not to use this data to allow, enable, or otherwise make possible,
dissemination or collection of this data, in part or in its entirety, for any
purpose, such as the transmission of unsolicited advertising and
and solicitations of any kind, including spam.  You further agree
not to use this data to enable high volume, automated or robotic electronic
processes designed to collect or compile this data for any purpose,
including mining this data for your own personal or commercial purposes. 

Please note: the registrant of the domain name is specified
in the "registrant" section.  In most cases, GoDaddy.com, LLC 
is not the registrant of domain names listed in this database.
