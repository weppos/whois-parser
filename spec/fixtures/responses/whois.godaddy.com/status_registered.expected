#domain
  %s == "godaddy.com"

#domain_id
  %s == "4013247_DOMAIN_COM-VRSN"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{1999-03-02 05:00:00 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2013-06-02 12:16:24 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2021-11-01 11:59:59 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "146"
  %s.name         == "GoDaddy.com, LLC"
  %s.organization == "GoDaddy.com, LLC"
  %s.url          == "http://www.godaddy.com"


#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == "Not Available From Registry"
  %s[0].name          == "Domain Administrator"
  %s[0].organization  == "Go Daddy Operating Company, LLC"
  %s[0].address       == "14455 N Hayden Rd Suite 219"
  %s[0].city          == "Scottsdale"
  %s[0].zip           == "85260"
  %s[0].state         == "Arizona"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.4805058800"
  %s[0].fax           == "+1.4805058844"
  %s[0].email         == "companynames@godaddy.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "Not Available From Registry"
  %s[0].name          == "Domain Administrator"
  %s[0].organization  == "Go Daddy Operating Company, LLC"
  %s[0].address       == "14455 N Hayden Rd Suite 219"
  %s[0].city          == "Scottsdale"
  %s[0].zip           == "85260"
  %s[0].state         == "Arizona"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.4805058800"
  %s[0].fax           == "+1.4805058844"
  %s[0].email         == "companynames@godaddy.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == "Not Available From Registry"
  %s[0].name          == "Domain Administrator"
  %s[0].organization  == "Go Daddy Operating Company, LLC"
  %s[0].address       == "14455 N Hayden Rd Suite 219"
  %s[0].city          == "Scottsdale"
  %s[0].zip           == "85260"
  %s[0].state         == "Arizona"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.4805058800"
  %s[0].fax           == "+1.4805058844"
  %s[0].email         == "companynames@godaddy.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{9}
  %s[0] %CLASS{nameserver}
  %s[0].name == "cns1.godaddy.com"
  %s[0].ipv4 == nil
  %s[0].ipv6 == nil
  %s[1] %CLASS{nameserver}
  %s[1].name == "cns2.godaddy.com"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil
  %s[2] %CLASS{nameserver}
  %s[2].name == "cns3.godaddy.com"
  %s[2].ipv4 == nil
  %s[2].ipv6 == nil
  %s[3] %CLASS{nameserver}
  %s[3].name == "a11-64.akam.net"
  %s[3].ipv4 == nil
  %s[3].ipv6 == nil
  %s[4] %CLASS{nameserver}
  %s[4].name == "a1-245.akam.net"
  %s[4].ipv4 == nil
  %s[4].ipv6 == nil
  %s[5] %CLASS{nameserver}
  %s[5].name == "a20-65.akam.net"
  %s[5].ipv4 == nil
  %s[5].ipv6 == nil
  %s[6] %CLASS{nameserver}
  %s[6].name == "a6-66.akam.net"
  %s[6].ipv4 == nil
  %s[6].ipv6 == nil
  %s[7] %CLASS{nameserver}
  %s[7].name == "a8-67.akam.net"
  %s[7].ipv4 == nil
  %s[7].ipv6 == nil
  %s[8] %CLASS{nameserver}
  %s[8].name == "a9-67.akam.net"
  %s[8].ipv4 == nil
  %s[8].ipv6 == nil
